collectd/files/plugin/collectd_zookeeper.py

```diff
@@ -25,6 +25,8 @@
 import collectd
 import socket
 
+# Default sampling interval
+INTERVAL = 50
```

Comment:
To comply with other plugins sampling interval instead of default 10s from /etc/collectd/collectd.conf